SU-SOE/stanford_soe_helper
# Collections: Label on Node style and field change

# READY FOR REVIEW

# Summary
This removes the block at the top of the article for the collection and puts the information under the date of the article.

# Needed By (Date)
- Sprint end

# Criticality
- Adds new layout for the mag article that is in a collection


# Steps to Test
- Install contrib module: https://www.drupal.org/project/colectomy to take care of colons on the display label
- Switch to this branch  -  7.x-2.x-SOE-2949 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2949)
- `drush cc all`
- `drush fr stanford_soe_helper_collection`
- `drush fr stanford_soe_helper_collection_layouts`
- go to: `admin/structure/types/manage/stanford-magazine-article/display/full`
- Move `This article is part of the series` field right below the `Byline` field. Make sure to link this field to the referenced entity and the label is set to `Inline no colon`.
- Save that display
- See: https://stanfordits.atlassian.net/browse/SOE-2949 for what to see.
- You can test off this page: `magazine/article/craig-criddle-redefining-waste-treatment`

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2949

## Related PRs

## More Information

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
